ui/src/app/applications/components/application-pod-view/pod-view.scss

```diff
@@ -0,0 +1,163 @@
+@import 'node_modules/argo-ui/src/styles/config';
+
+$pod-size: 25px;
+$gutter: 3px;
+$pods-per-row: 8;
+$max-rows: 5;
+
+.pod-view {
+    &__settings {
+        align-items: center;
+        display: flex;
+        margin-bottom: 1em;
+        &__section {
+            margin: 0 10px;
+        }
+    }
+}
+
+.nodes-container {
```

Comment:
Please make `nodes-container` and `node` sub-classes of `pod-view` . E.g. `pod-view__nodes-container` and `pod-view__node`